src/main/java/com/redhat/devtools/lsp4ij/features/selectionRange/LSPExtendWordSelectionHandler.java

```diff
@@ -33,31 +33,23 @@
 /**
  * Implementation of the IDE's extendWordSelectionHandler EP for LSP4IJ files against textDocument/selectionRange.
  */
-public class LSPExtendWordSelectionHandler implements ExtendWordSelectionHandler {
+public class LSPExtendWordSelectionHandler extends AbstractLSPExtendWordSelectionHandler {
```

Comment:
Made the original `extendWordSelectionHandler` implementation extend the new shared abstract base class.